src/python/pants/backend/python/tasks/pex_build_util.py

```diff
@@ -127,6 +127,15 @@ def dump_requirements(builder, interpreter, reqs, log, platforms=None):
       locations.add(dist.location)
 
 
+def subsystems_used():
```

Comment:
This hits 4 tasks and 1 subsystem in addition to python_binary_create.py. I'd be happy with all of these being fixed in one PR but would also be happy with an issue to track the wider fix.